foundry.toml

```diff
@@ -4,7 +4,10 @@
   # auto-detect solc versions.
   optimizer = true
   optimizer_runs = 10_000_000
-  remappings = ["openzeppelin-contracts/=lib/openzeppelin-contracts/contracts"]
+  remappings = [
+    "@openzeppelin/contracts/=lib/openzeppelin-contracts/contracts",
+    "openzeppelin-contracts/=lib/openzeppelin-contracts/contracts",
```

Comment:
I think it'd be better to change all imports to the `@openzeppelin` style instead of just a subset and having two remappings?

Reply:
Fair. Done in [c046e6d](https://github.com/ScopeLift/flexible-voting/pull/45/commits/c046e6df1faaca603e57518fdf9bc7297b911498)